docs/source/index.md

```diff
@@ -0,0 +1,98 @@
+# Getting Started with jeta
+
+## What is jeta?
+
+The `jeta` package is an API for monitoring and accessing FOS Engineering Telemetry Archive. The API exposes functions for ingesting/processing telemetry (tlm),
+archive status/managment, and data fetching. Tools are built around the python data science stack (i.e. numpy, pandas, matplotlib ... etc.) 
+and well as HDF5 standard and supporting python implementations (h5py, pytables). There are multiple ways to access and use the jeta package 
+including directly after installing with `pip`, inside a pre-built enviroment with jupyter lab, a web API using HTTP request, and a web application
+that provides a front-end for the most common use cases for the tools.
+
+The package, `jeta` is a LITA functional-equivalent of [`Ska.eng_archive`](https://github.com/sot/eng_archive).
+[Ska](https://cxc.cfa.harvard.edu/mta/ASPECT/tool_doc/pydocs/) is the "engineering telemetry archive is a suite of tools and data products" for
+that supports the [Chandra](https://chandra.harvard.edu/about/spacecraft.html) X-Ray Observatiory. Where as `jeta` fulfills a similar role for JWST.
+The package is still dependant on the Chandra package environment, [skare3](https://github.com/sot/skare3).
+
+## Installing `jeta`
```

Comment:
not sure what this will look like in it's final form, but maybe move these installation instructions to the end of the page and emphasize that it is not needed for typical use cases.